argocd/resource_argocd_repository_test.go

```diff
@@ -99,6 +154,34 @@ resource "argocd_repository" "helm" {
 `)
 }
 
+func testAccArgoCDRepositoryHelmProjectScoped(project string) string {
+	return fmt.Sprintf(`
+	resource "argocd_project" "simple" {
+	metadata {
+		name      = "%s"
+		namespace = "argocd"
+	}
+
+	spec {
+		description  = "simple project"
+		source_repos = ["*"]
+
+		destination {
+		name      = "anothercluster"
+		namespace = "bar"
+		}
+	}
+	}
+
+resource "argocd_repository" "helm" {
+  repo = "https://helm.nginx.com/stable"
+  name = "nginx-stable-scoped"
+  type = "helm"
+  project = "%s"
+}
+`, project, project)
+}
```

Comment:
Nit: formatting + use of format indexing to ensure resulting string is always correct.

``` suggestion
resource "argocd_project" "simple" {
  metadata {
    name      = "%[1]s"
    namespace = "argocd"
  }

  spec {
    description  = "simple project"
    source_repos = ["*"]
    
	destination {
      name      = "anothercluster"
      namespace = "bar"
    }
  }
}

resource "argocd_repository" "helm" {
  repo = "https://helm.nginx.com/stable"
  name = "nginx-stable-scoped"
  type = "helm"
  project = "%[1]s"
}
`, project)
}
```

Reply:
didn't know the indexing syntax, nice :)